wasm/src/main/scala/org/scalajs/linker/backend/wasmemitter/Emitter.scala

```diff
@@ -247,7 +258,7 @@ final class Emitter(config: Emitter.Config) {
     // Finish the start function
 
     fb.buildAndAddToModule()
-    ctx.moduleBuilder.setStart(genFunctionName.start)
+    // ctx.moduleBuilder.setStart(genFunctionName.start)
```

Comment:
[WASI application ABI]( https://github.com/WebAssembly/WASI/blob/main/legacy/application-abi.md) requires to export `start` or `initialize` function, and WASI will setup the runtime in there (I suppose).

If we set `start` function for the module, Wasm module will call the `start` function when we instantiate it. If `start` function calls a WASI function (in `main` method in Scala) it will fail because WASI haven't yet prepared.